pilot/pkg/bootstrap/server.go

```diff
@@ -308,6 +308,7 @@ func NewServer(args *PilotArgs, initFuncs ...func(*Server)) (*Server, error) {
 	// is used as the authentication result.
 	authenticators := []security.Authenticator{
 		&authenticate.ClientCertAuthenticator{},
+		&authenticate.CidrAuthenticator{},
```

Comment:
It can be any where now. If it is configured to use the Cidrs it will be used otherwise it is not. Having said that, I can move it to end if you prefer that